tests/integration/test_end_to_end_election.py

```diff
@@ -425,7 +425,7 @@ def publish_results(self) -> None:
             self.constants,
             [self.device],
             self.ballot_store.all(),
-            self.ciphertext_tally.spoiled_ballots.values(),
+            self.plaintext_tally.spoiled_ballots.values(),
```

Comment:
This isn't passing because the objects don't line up as 1:1 as they should. The `PlaintextTally` has a dictionary of contests by ballot id where the `PlaintextBallot` is an array of those same contests. 

Relevant lines:
https://github.com/microsoft/electionguard-python/blob/b09fc6465d3a4b8acea0f9e06ef304a539fc4e61/src/electionguard/tally.py#L177
https://github.com/microsoft/electionguard-python/blob/35a76a43cbe08515f94fd69b9637555699555978/src/electionguard/ballot.py#L586

That being said, the need is still real. This can be solved within this PR without dragging in too much scope by making a helper function.

Create a function in `ballot.py` like:
```
def get_spoiled_ballots_from_plaintext_tally(tally: PlaintextTally) -> PlaintextBallot[]:
```

OR


Create a function in `tally.py` like:
```
def create_plaintext_ballot_from_contests(ballot_id: str, contests: PlaintextContest[], ballot_style: str ="") -> PlaintextBallot
```
The `ballot_style` should be an empty string. It's not available at this point in the process.

Once one of these is made, a conversion can be done right before this line to ensure it's the correct object type then the tests should pass.